Docker/Dockerfile

```diff
@@ -9,24 +9,34 @@ RUN apt-get update && apt-get install -q -y --no-install-recommends \
     dirmngr \
     gnupg2 \
     lsb-release \
+    wget \
     python3-colcon-ros \
     && apt-get clean
 
 RUN mkdir -p /home/ros2_ws/src \
     && cd /home/ros2_ws/src/ \
-    && git clone https://github.com/ros-simulation/gazebo_ros2_control \
+    # && wget https://raw.githubusercontent.com/ros-planning/moveit2/main/moveit2.repos \
+    # && vcs import < moveit2.repos \
+    && git clone https://github.com/ros-planning/moveit2 \
```

Comment:
I think we can remove all these moveit demos now that the demos will live in https://github.com/ros-simulation/gazebo_ros_demos/pull/41 right?